webapp/src/apis/auth-api.test.js

```diff
@@ -0,0 +1,16 @@
+import AuthApi from './auth-api'
+
+beforeEach(() => {
+  fetchMock.mockClear()
+  fetchMock.doMock()
+})
+
+it('Test Auth login', async () => {
```

Comment:
Should we include the fail/non-200 case in here to show what happens then?

Reply:
I just added a test for this!